active_3d_planning_voxblox/src/map/uncertainty_aware_map.cpp

```diff
@@ -0,0 +1,99 @@
+#include "active_3d_planning_voxblox/map/uncertainty_aware_map.h"
+
+#include <voxblox_ros/ros_params.h>
+
+#include "active_3d_planning_core/data/system_constraints.h"
+
+namespace active_3d_planning {
+namespace map {
+
+ModuleFactoryRegistry::Registration<UncertaintyAwareMap>
+    UncertaintyAwareMap::registration("VoxbloxMapWithUncertainty");
+
+UncertaintyAwareMap::UncertaintyAwareMap(PlannerI& planner)
+    : VoxelValueMap(planner), voxbloxMap(planner) {}
+
+voxblox::EsdfServer& UncertaintyAwareMap::getESDFServer() {
+  return voxbloxMap.getESDFServer();
+}
+
+void UncertaintyAwareMap::setupFromParamMap(Module::ParamMap* param_map) {
+  // Reconstruction + collision map
+  voxbloxMap.setupFromParamMap(param_map);
+
+  // Uncertainty Map
+  ros::NodeHandle nh("/uncertainty/map/");
```

Comment:
Avoid global namespaces, this namespace could e.g. be a param of the map.

Reply:
cc32191